frontend/src/libs/components/page-layout/page-layout.tsx

```diff
@@ -40,7 +40,7 @@ const PageLayout: FC<Properties> = ({
       )}
       <main className={styles.content}>
         <RouterOutlet />
-        {children}
+        {typeof children === 'function' ? children(selectedTab) : children}
```

Comment:
why do you need it?

Reply:
I was mocking for the demo, we don’t need it